test/controllers/EligibilityCalcControllerTest.scala

```diff
@@ -22,7 +22,7 @@ import play.api.test.FakeRequest
 import play.api.test.Helpers.{contentAsString, defaultAwaitTimeout}
 import utils.ControllerBaseTest
 
-
+//TODO: Update values in test names
```

Comment:
Can you please remove the comment or is it needed?

Reply:
Please can we keep it in? There are some test descriptions that need updating. Dan and i wanted to raise it as a backlog ticket